### PR TITLE
Fix typo in adapter of webhookCallback

### PR DESCRIPTION
**Pull request by Hth4nh**
Change "std/http" to "https"

### CODE REVIEW

**Comment by LWJerri** [comment]
Related to [this](https://t.me/grammyjs/283555) and [this](https://t.me/grammyjs/278208).

File: site/docs/hosting/vercel.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ if (!token) throw new Error("BOT_TOKEN is unset");
 
 const bot = new Bot(token);
 
-export default webhookCallback(bot, "std/http");
+export default webhookCallback(bot, "https");
```
Comment (LWJerri) [inline site/docs/hosting/vercel.md:83]: Do the same for another language, please.

Comment (Hth4nh) [reply site/docs/hosting/vercel.md:83]: Done. Pls check.

**Review by LWJerri — APPROVED** [review]

**Review by KnorpelSenf — APPROVED** [review]
Thanks!

**Review by rojvv — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**